src/buffer/out/textBuffer.cpp

```diff
@@ -1289,6 +1310,13 @@ bool TextBuffer::MoveToPreviousWord(COORD& pos, std::wstring_view wordDelimiters
     auto copy = pos;
     auto bufferSize = GetSize();
 
+    // GH#7663: Treat EndExclusive as EndInclusive so
+    // that it actually points to a space in the buffer
+    if (pos == bufferSize.EndExclusive())
+    {
+        copy = { bufferSize.RightExclusive(), bufferSize.BottomInclusive() };

```

Comment:
I am worried that this is actually another version of "end exclusive" -- the invalid one. To be inclusive, it would need to be both right AND bottom inclusive, no?

Reply:
it has the same comment as above (!!!!) but a different implementation (!!!!)

Reply:
Good catch